docs/reference/method/MongoDBDatabase-createCollection.txt

```diff
@@ -296,7 +296,7 @@ Parameters
          rules or expressions. You can specify the expressions using the same
          operators as MongoDB's
          :manual:`query operators </reference/operator/query>` with the
-         exception of :query:`$geoNear`, :query:`$near`, :query:`$nearSphere`,
+         exception of :query:`$near`, :query:`$nearSphere`,
```

Comment:
I assume this dates back to some earlier version of the docs (e.g. 3.2), but I confirmed `$geoNear` is no longer mentioned in the [create](https://www.mongodb.com/docs/manual/reference/command/create/) docs.